<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Überprüfen</string>
   </property>
  </widget>
  <widget class="QLabel" name="answer_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1-b, 2-c, 3-e, 4-a, 5-f, 6-d, 7-h, 8-j, 9-k, 10-g</string>
   </property>
  </widget>
  <widget class="QLabel" name="answer_label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>11-i, 12-m, 13- n, 14-l, 15-q, 16-o, 17-p, 18-s, 19-t, 20-r</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>answer_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>answer_label_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
